core/domain/stats_jobs_one_off.py

```diff
@@ -802,6 +802,20 @@ def entity_classes_to_map_over(cls):
     @classmethod
     def require_non_negative(
             cls, exp_id, exp_version, property_name, value, state_name=None):
+        """Yields the Negative count of the exploration according
+        to the passed property name.
+
+        Args:
+            exp_id: str. Exploration ID.
+            exp_version: str. Exploration version.
+            property_name: str. Property name.
+            value: dict. Values of different properties for the exploration.
+            state_name: str. State name of the exploration.
```

Comment:
The state name to include in the error message.

(There is no such thing as a "state name of an exploration". An exploration has many different states.)